neurons/miner.py

```diff
@@ -167,16 +168,25 @@ def generate_prediction(self, historical_prices):
 
     async def forward(self, synapse: crypto_ai.protocol.Dummy) -> crypto_ai.protocol.Dummy:
         requested_symbols = synapse.request_data.get('symbols', self.crypto_symbols)
+        requested_intervals = synapse.request_data.get('intervals', self.intervals)
         requested_currency = synapse.request_data.get('currency', self.currency)  # Use request currency or default
 
-        is_valid = any((len(symbol) <= 5 or len(symbol) >= 2) for symbol in requested_symbols)
-
-        if not is_valid:
+        # check if provided symbols are supported by the miner
+        is_valid_symbol = all(symbol in self.crypto_symbols for symbol in requested_symbols)
+        if not is_valid_symbol:
             msg = "No supported crypto symbols provided"
             synapse.dummy_output = msg
             bt.logging.warning(msg)
             return synapse
 
+        # check if provided intervals are supported by the miner
+        is_valid_interval = all(interval in self.intervals for interval in requested_intervals)
+        if not is_valid_interval:
+            msg = "No supported intervals provided"
+            synapse.dummy_output = msg
+            bt.logging.warning(msg)
+            return synapse
```

Comment:
we could add an error field in the synapse so that the validator server can also see the issue

Reply:
test

Reply:
test

Reply:
Done.